wuzz.go

```diff
@@ -1188,6 +1188,31 @@ func (a *App) ParseArgs(g *gocui.Gui, args []string) error {
 			}
 		case "-k", "--insecure":
 			a.config.General.Insecure = true
+		case "-T", "--tls":
+			if arg_index >= args_len-1 {
+				g.Close()
+				log.Fatal("Missing TLS version range: MIN,MAX")
```

Comment:
Please rewrite to return with an error to break argument parsing instead of the above two lines.